spec/schemas/_common.yaml

```
@@ -20,137 +22,164 @@ components:
       required:
         - acknowledged
     DocStatus:
-      description: |-
-        The item level REST category class codes during indexing.
+      description: The item level REST category class codes during indexing.
```

```suggestion
      description: The item level HTTP response status code during indexing.
```

@nhtruong: Per your comment, do you think this description is better?

---

Yes. That's more accurate :)